### ISSUE TITLE
npm audit issue with lodash.clonedeep

### ISSUE THREAD

**Issue by jpike88** [lodash/lodash#4318]
```npm i lodash.clonedeep```
```npm audit```
```
 Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ 254e79efcf38b9e25524af14d794de2fef1ab95ec262de66a4d9d83fcb8… │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ 254e79efcf38b9e25524af14d794de2fef1ab95ec262de66a4d9d83fcb8… │
│               │ > lodash                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/782    
```

**Comment by jpike88** [comment]
I just noticed that the last time per-method packages were released on npm.... 3 years ago??? Why wouldn't you have an auto-publish on those each time you're releasing the main lodash package? That's a security issue waiting to happen.

**Comment by jdalton** [comment]
Hi @jpike88!

I'm not really maintaining the individual method packages. We'll bump them eventually as part of one last 4.x push for security fixes but I'm in no hurry.

**Comment by jpike88** [comment]
weird, I can't seem to get rid of the warning even after switching to the main lodash repo

```
Joshs-Mac-mini-2:server joshpike$ npm i lodash
npm WARN fp-backend@1.0.0 No description
npm WARN fp-backend@1.0.0 No repository field.

+ lodash@4.17.11
updated 1 package and audited 2117 packages in 2.669s
found 1 moderate severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
Joshs-Mac-mini-2:server joshpike$ npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ f214d1b2191274276fb4b6e6f69d1860abfe593edfeadf888b5539b3914… │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ f214d1b2191274276fb4b6e6f69d1860abfe593edfeadf888b5539b3914… │
│               │ > lodash                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/782                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 moderate severity vulnerability in 2117 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```
